flow/data_pipeline/query.py

```diff
@@ -179,6 +200,41 @@ class QueryStrings(Enum):
                                                                          'POWER_DEMAND_MODEL_DENOISED_ACCEL_VEL',
                                                                          'denoised_speed_cte'))
 
+    FACT_SAFETY_METRICS = """
+        SELECT
+            vt.id,
+            vt.time_step,
+            COALESCE((
+                value_lower_left*(headway_upper-headway)*(rel_speed_upper-leader_rel_speed) + 
+                value_lower_right*(headway-headway_lower)*(rel_speed_upper-leader_rel_speed) + 
+                value_upper_left*(headway_upper-headway)*(leader_rel_speed-rel_speed_lower) + 
+                value_upper_right*(headway-headway_lower)*(leader_rel_speed-rel_speed_lower)
+            ) / ((headway_upper-headway_lower)*(rel_speed_upper-rel_speed_lower)), 200) AS safety_value,
+            vt.source_id
+        FROM fact_vehicle_trace vt
+        LEFT OUTER JOIN fact_safety_matrix sm ON 1 = 1
```

Comment:
I assume `fact_safety_matrix` is referring to the CSV file you added on S3, is there any updates or maintenance I should perform on it?